docs/adr/0002-caching.md

```diff
@@ -0,0 +1,24 @@
+# 2. Caching
+
+Date: 2021-10-05
+
+## Status
+
+Accepted
+
+## Context
+
+- Client always make requests to back-end to refresh the data
+- User need to wait a long time to see the mailbox and email
```

Comment:
Always requesting the same entities again and over again put a lot of pressure on the backend.